core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-page-tools-seo/dot-page-tools-seo.component.spec.ts

```diff
@@ -40,12 +40,17 @@ describe('DotPageToolsSeoComponent', () => {
         spectator = createHost(
```

Comment:
You don't need to create a host to pass custom inputs to the component, you can use the component factory and then create the component with the needed inputs, check here: https://github.com/dotCMS/core/pull/25600/files#diff-71156e8bc7aa2247cecab3f2627182871f930b385d4ff4de19df2f0951667597R43